io.github.syco.blink.json

```diff
@@ -0,0 +1,384 @@
+      ]
+    },
+    {
+      "name": "python-xcaplib",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.ag-projects.com/XCAP/python-xcaplib-1.2.2.tar.gz",
+          "sha256": "d60ce7078ce08bcb8033154ba293aaba19df66d0728889e4bfff1d1d876e76c8"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "python2 setup.py install"
+      ]
+    },
+    {
+      "name": "python-msrplib",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.ag-projects.com/MSRP/python-msrplib-0.20.0.tar.gz",
+          "sha256": "3a2ed4f174b6e758bb511b05211177cdc0c887c77488cbf0c31fe34b669d6231"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "python2 setup.py install"
+      ]
+    },
+    {
+      "name": "python-cjson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/AGProjects/python-cjson/archive/release-1.2.2.tar.gz",
+          "sha256": "dacb2206b7d5d0fdf8d2901ed9a4c3994b8a3fc65d82f100ed5535e43e174452"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "python2 setup.py install"
+      ]
+    },
+    {
+      "name": "blink",
+      "modules": [
+        {
+          "name": "LibVNC",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/LibVNC/libvncserver/archive/LibVNCServer-0.9.12.zip",
+              "sha256": "4ca21c6c2ba78ebccbfef8b35b7cd990bd430ab24f813b17c4e33e5f1f7838b8"
+            }
+          ],
+          "buildsystem": "cmake",
+          "config-opts": [
+            "-DCMAKE_BUILD_TYPE=Release"
+          ]
+        }
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/AGProjects/blink-qt/archive/release-3.2.1.tar.gz",
+          "sha256": "d1987a15c267fd6439b026e970e13fa7aa7ada3ac4f69aa6cf36fb201b57dcd8"
+        },
+        {
+          "type": "file",
+          "path": "files/blink.desktop",
+          "sha256": "91e19391f021616ea60744fa2cabf988cc82c90e672b4f0a7d065cb761b9011c"
+        },
+        {
+          "type": "file",
+          "path": "files/blink.png",
+          "sha256": "c30c6a66bed48d65c17d7ed68303ccc87710d8ac35645e83ac51d7985f4b5ae9"
+        },
+        {
+          "type": "file",
+          "path": "files/blink.xml",
+          "sha256": "0aea3a410f4a13d0288f6b4c1fd0664275e3c0ef0cae8b0a16b664c9b7fb240f"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "python2 setup.py build_ext",
+        "python2 setup.py install"
+      ],
+      "post-install": [
+        "install -Dm644 blink.desktop /app/share/applications/blink.desktop",
+        "install -Dm644 blink.png /app/share/icons/io.github.syco.blink.png",
+        "install -Dm644 blink.xml /app/share/appdata/io.github.syco.blink.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html